pkgs/development/haskell-modules/configuration-ghcjs.nix

```diff
@@ -79,12 +79,6 @@ self: super: {
   ghcjs-dom = overrideCabal super.ghcjs-dom (drv: {
     buildDepends = [ self.base self.mtl self.text self.ghcjs-base ];
```

Comment:
`ghcjs-dom` depends on different libraries depending on whether we are compiling with GHC or GHCJS. Here we make the GHCJS dependencies explicit, otherwise nix will try to compile stuff like GTK or GLIB, and that will fail.